turbo.json

```diff
@@ -37,7 +32,7 @@
     },
     "zip": {
       "dependsOn": [
-        "^build"
+        "build"
```

Comment:
Why you remove it from here, but in the above ```build``` it's still?
Where's the difference, it's between configs in this package and the rest?
I wasn't configuring turbo before, and i don't know if it's really necessary 😃

Reply:
Actually this is pure experimental finding, to make `zip:firefox` and `zip` properly build corresponding dist (with proper manifest).

It was not working with ^build, I can only assume that it doesn't send down environment variables to the children build scripts.